Add files PLM

frpalmer opened the pull request:
Add logo and info

frpalmer committed:
Add files PLM

Add logo and info

frpalmes commented:
Fee paid: https://explorer.binance.org/tx/46E67E083E5CCA3F6DB074DA60DE7BF12E513B59524258C788E1FF8EF6328294

daihovey committed:
Add TAPE (#2126)

vikmeup committed:
Daily update

coreycaplan3 committed:
Add DMG Token (#2122)

* Added DMM dapp, tokens, and added tokens to whitelist

* Adjusted image sizes to match spec

* updated whitelist as well as DMG token

* Fixed DMG background

Commit:
Krios/GIG Token (GIG)

kolya182 committed:
Daily update

abhisingla280 committed:
Add DAWN Token (#2131)

Contract Address: 0x580c8520dEDA0a441522AEAe0f9F7A5f29629aFa
Website: https://dawn.org/
Etherscan: https://etherscan.io/token/0x580c8520dEDA0a441522AEAe0f9F7A5f29629aFa
GitHub: https://github.com/Dawn-Protocol
Discord: https://discord.gg/MgV9YDu
Contact: team@dawn.org

kolya182 committed:
Daily update

kolya182 committed:
Update validators info

soliditycdr committed:
Please add YCN on TrustWallet (#2132)

Token Name	: YouthCoin
Symbol		: YCN
Decimals	: 8
Total Supply	: 350,000,000 YCN
Contract Address: 0xD9D2C606EC5F7a01dF496768cfC9E5003B23d193
Original Logo:

kolya182 committed:
Daily update

akru committed:
Add Robonomics (XRT) (#2134)

kolya182 committed:
Daily update

frpalmer committed:
Merge branch 'master' into frpalmes-PLM
